test/node/features/connect.feature

```diff
@@ -22,16 +22,18 @@ Scenario: a synchronous thrown error in a route
   And the event "request.httpMethod" equals "GET"
 
 Scenario: an asynchronous thrown error in a route
-  Then I open the URL "http://connect/async"
+  Then I open the URL "http://connect/async" tolerating any error
```

Comment:
tolerating any error is not required because the node process is allowed to crash

Reply:
Do you mean "is now required"?